.github/workflows/clean-repo.yml

```
@@ -0,0 +1,40 @@
+name: "clean repo"
+
+on:
+  schedule:
+    - cron: "0 1 1 * *" # Runs at 01:00, on day 1 of the month
+
+jobs:
+  clean-repo:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+      # Call clean repo
+      - name: Clean repo
+        id: clean-repo-step
+        uses: dotnet/docs-tools/cleanrepo/CleanRepo@main
+        with:
+          start-directory: '.'
+          docset-root: './docs'
+          repo-root: '.'
+          delete: true
+          orphaned-topics: true
+          orphaned-images: true
+          orphaned-snippet: true
+          orphaned-includes: true
+          replace-redirects: true
+          relative-links: true
+          remove-hops: true
+        
+      # Create the PR for the work done by the "clean repo" tool
+      - name: create-pull-request
+        uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04
+        with:
+          title: "Monthly chores: automated clean repo tool"
```

based on what we learned with the what's new tool, let's declare a unique branch here:

```suggestion
          branch: create-cleanrepo-pull-request/patch
          title: "Monthly chores: automated clean repo tool"
```